pkg/driver_cryptocell/include/psa_error.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2021 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_psa_crypto pkg_driver_cryptocell
+ * @{
+ *
+ * @file
+ * @brief       Glue code translating between PSA Crypto and the CryptoCell 310 driver APIs
+ *
+ * @author      Lena Boeckmann <lena.boeckmann@haw-hamburg.de>
+ *
+ */
+
+#ifndef PSA_ERROR_H
+#define PSA_ERROR_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "psa/crypto.h"
+#include "crys_hash_error.h"
+#include "ssi_aes_error.h"
+
+/**
+ * @brief   Convert CryptoCell CRYS errors to PSA status values
+ *
+ * @param   error CRYSErrot_t
+ * @return  @ref psa_status_t
```

Comment:
Doxygen is: "unable to resolve reference to 'psa_status_t' for ref command" there might be a || DOXYGEN missing in some #ifdef

Reply:
I think its defined in `sys/include/psa/crypto_types.h` L338 which is part of  #18578

Reply:
Yeah, I haven't really done anything with doxygen so far and I'm trying to figure out how to cross reference over multiple files. 
Would it be better to remove all references until everything is merged and then add them later?

Reply:
hopefully it will solve when https://github.com/RIOT-OS/RIOT/pull/18578 is merged if not a exception_pattern for that needs to be added to doccheck